### ISSUE TITLE
Styling issues

### ISSUE THREAD

**Issue by Daltonstudio** [juho-jaakkola/elgg-tour#2]
A few minor things, don't know if it's due to the plugin, joyride or my theme. 
- The container and the arrow would be better if they were on top of the white glow.
- The next button doesn't go to a new line.
- The container could use soms more padding-right so that it has a little spacing between the 'closing x' when not using a title.

![Image of Yaktocat](http://driesdk.be/junk/tourstyle.jpg)

**Comment by juho-jaakkola** [comment]
Were you using editor (TinyMCE, CKEditor) when inserting the content?

**Comment by Daltonstudio** [comment]
Nope, but that explains it probably. Maybe I should manually add a line break.

**Comment by juho-jaakkola** [comment]
The contents will be automatically wrapped inside `<p>` tags when using an editor. But I think I'll add some padding by default anyway. Cannot expect all sites to use an editor.

**Comment by Daltonstudio** [comment]
Great, it's not a problem though to add and extra opening and closing paragraph tag in the field, but it should be noted somewhere then for people not using an editor.

**Comment by juho-jaakkola** [comment]
Could you suggest a `z-index` value that would fix the glow issue? (Preferably tested in default, Aalborg and your own theme.)

**Comment by Daltonstudio** [comment]
I dont't think it's a z-index issue anymore, it's just that the container is too much up and due to it's transparency it gives a strange result where you see the element trough the arrow.
It seems like (this is on default aalborg), that the first time I load the tour all is right and the tourstop is loaded like it should, but on a second load at the same page (without a page refresh, just close the tour and start it again), the container is positioned a few pixels more up.
Do you have the same issue? I can't try on a fresh install right now.

![second load](http://driesdk.be/junk/secondload.jpg)

**Comment by juho-jaakkola** [comment]
I see, so the info box itself is slightly transparent.

Some alternatives for the shadow:
- Use inner shadow
- Use border (may beak layouts because adds width to element)
